src/TrailerPreview/TrailerPreview.js

```diff
@@ -1,34 +1,52 @@
-import React, { Component } from 'react';
+import React, { Component, useState } from 'react';
 import ReactPlayer from 'react-player/youtube';
 import { getMovieVideoData } from '../apiCalls';
 import './TrailerPreview.scss';
 
 class TrailerPreview extends Component {
-  constructor({ trailerInfo }) {
+  constructor() {
     super();
     this.state = {
       trailerInfo: '',
       trailerId: '',
     };
   }
 
-  // TODO: Sad path, divert to generic video if no trailer
+  //737173
+  //this.props.trailerInfo.id
+
   componentDidUpdate = (prevProps) => {
     if (this.props.trailerInfo !== prevProps.trailerInfo) {
       getMovieVideoData(this.props.trailerInfo.id).then((data) =>
         this.setState({
           trailerInfo: data.videos,
-          trailerId: data.videos.find((video) => video.type === 'Trailer').key,
+          trailerId:
+            // '2Gg6Seob5Mg' ||
+            data.videos.find((video) => video.type === 'Trailer').key,
         })
       );
+      if (!this.state.trailerId) this.setState({ trailerId: '2Gg6Seob5Mg' });
     }
   };
 
   render() {
+    const {
+      // backdrop_path,
+      // poster_path,
+      title,
+      release_date,
+      // overview,
+      genres,
+      // budget,
+      // revenue,
+      // tagline,
+      runtime,
+    } = this.props.trailerInfo;
+
     return (
       <section>
-        <div className="player-wrapper">
-          {this.state.trailerId && (
+        {this.state.trailerId && (
```

Comment:
This looks good, glad you added a feature that allows users to just click on a whole trailer that takes them to a movie preview